src/layout/MainLayout/Header/ProfileSection/index.js

```diff
@@ -21,31 +18,27 @@ import {
   Stack,
   Typography
 } from '@mui/material';
-
-// third-party
 import PerfectScrollbar from 'react-perfect-scrollbar';
-
-// project imports
 import MainCard from 'ui-component/cards/MainCard';
 import Transitions from 'ui-component/extended/Transitions';
 import User1 from 'assets/images/users/user-round.svg';
-
-// assets
-import { IconLogout, IconSettings, IconUser } from '@tabler/icons';
-
-// ==============================|| PROFILE MENU ||============================== //
+import { IconLogout, IconSettings } from '@tabler/icons';
 
 const ProfileSection = () => {
   const { logout } = useAuth();
   const theme = useTheme();
-  const customization = useSelector((state) => state.customization);
+  const customization = useSelector((state) => state.customization || { borderRadius: 8 });
   const navigate = useNavigate();
-
   const [selectedIndex, setSelectedIndex] = useState(-1);
   const [open, setOpen] = useState(false);
-
+  const {username} = useSelector((state) => state.user.user || "User");
```

Comment:
Instead of using three Selectors, get the user from the useSelector and use the object

const { user } = useSelector((state) => state.user);

{user.username || "xyz"}